go/enclave/components/rollup_consumer.go

```diff
@@ -159,11 +159,13 @@ func (rc *rollupConsumerImpl) extractAndVerifyRollups(processed *common.Processe
 			rc.logger.Warn(fmt.Sprintf("could not decode tx at index %d. Cause: %s", i, err))
 		}
 		if t == nil {
+			rc.logger.Debug(fmt.Sprintf("rollup event was provided with a tx that is not a rollup."))
 			continue
 		}
 
 		rollupHashes, ok := t.(*common.L1RollupHashes)
 		if !ok {
+			rc.logger.Debug(fmt.Sprintf("could not cast decoded transaction to l1 rollup hashes. Cause: %s", err))
```

Comment:
Slightly prefer `.Debug("could not cast decoded transaction to l1 rollup hashes", log.ErrKey, err)

but I'm guessing AI put in those Sprintf's because it saw some others in the file already so fair enough haha.